src/Pages/yoonseonghwang/Main/Main.js

```diff
@@ -1,8 +1,133 @@
 import React from 'react';
+import './Main.scss';
+import Nav from '../../../Components/Nav/Nav';
+import Feed from './Component/Feed';
 
 class Main extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      feedSrc: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/commentData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          feedSrc: data,
+        });
+      });
```

Comment:
```suggestion
  componentDidMount() {
    fetch('/data/commentData.json')
      .then(res => res.json())
      .then(({ feedSrc }) => this.setState({ feedSrc }));
```
- public 경로에서 가져올 경우 url과 포트 번호를 생략해줄 수 있습니다
- get method는 기본 값입니다!
- 객체 단축 속성명을 통해 같은 이름의 키와 벨류는 합쳐써줄 수 있습니다